client/app/bundles/course/virtual_classrooms.js

```diff
@@ -25,13 +39,72 @@ $(document).ready(() => {
         $this.removeClass('disabled');
         const { response: { data: { errors } } } = error;
         if (errors) {
-          $this.closest('.access-link')
-            .append('<br/>')
-            .append($('<i></i>')
-              .addClass('error')
-              .text(errors)
+          $this.closest('.access-link').append($('<br/>'));
+          appendError($this, '.access-link', errors);
+        }
+      });
+  });
+
+  $(document).on('click', '.fetch-recorded-videos', function () {
+    const $this = $(this);
+    const $div = $this.closest('.recorded-videos');
+    const splits = $this.attr('id').split('-');
+    const virtualClassroomId = splits[2];
+    $this.addClass('disabled');
+
+    CourseAPI.virtualClassrooms.recordedVideos(virtualClassroomId)
+      .then((res) => {
+        const { data: arr } = res;
```

Comment:
also here, do you have a better name instead of `arr` and `rec`?  I believe it will make the code more readable.